pom.client.xml

```diff
@@ -113,7 +113,8 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.8.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
+        <version>3.8.1
+        </version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
```

Comment:
What's up with the line wrapping?

Reply:
IntelliJ auto formatting breaking lines after 120 characters, will update my configuration to ignore XML files (or set it to something large like 200 characters).